### PR TITLE
Add issue marker for extension point reservation discussion

### PR DESCRIPTION
**Pull request by OR13**
Screen Shot 2023-06-06 at 1 20 24 PM

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1141.html" title="Last updated on Jun 21, 2023, 1:13 PM UTC (6636dff)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1141/e6e9fbf...6636dff.html" title="Last updated on Jun 21, 2023, 1:13 PM UTC (6636dff)">Diff</a>

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]

**Comment by TallTed** [comment]
"Holder binding."  The misnomer that will not die.

**Review by awoie — APPROVED** [review]

**Review by msporny — CHANGES_REQUESTED** [review]
I thought the WG agreed to reserve `confidenceMethod` and point to the CCG work item? This issue marker deviates from how all of the other properties are being reserved (in that, `confidenceMethod` is not being reserved). 

Specific changes requested, align the way `confidenceMethod` is being reserved to the same way that the other properties are being reserved. (or, specify why `confidenceMethod` is being treated differently).

**Comment by decentralgabe** [comment]
agree with @msporny I thought we had consensus on `confidenceMethod`. is there an anti-reserve list 🙃

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-06-14-vcwg#section1-1) on 2023-06-14

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.1. Add issue marker for holder binding (pr vc-data-model#1141)

_See github pull request [vc-data-model#1141](https://github.com/w3c/vc-data-model/pull/1141)._




**Michael Prorock:** suggest we merge first, bikeshed naming and properties second.  

**Dave Longley:** this is not bike shedding, its data modeling... in the information graph.  
… id is a special property that has specific behavior.  
… because of JSON-LD this could impact selective disclosure.  
… its not just about picking a name.  

**Kristina Yasuda:** ask for a special topic call if you need it.  

**Manu Sporny:** its not bikeshedding, there are design considerations that effect JSON-LD selective disclosure.  
… the SRI format is different, its not what SRI does... we should either align with SRI or align with something else.  
… there are design considerations that effect selective disclosure in JSON-LD nquads.  

> *Ivan Herman:* See [Subresource integrity spec:](https://www.w3.org/TR/SRI/).

**Manu Sporny:** I am reviewing the PR now.  
… people are requesting discussion.  

**Orie Steele:** You're open to open a PR to follow best practice.  

> *Orie Steele:* AFAIK there is no bar, right now.

> *Orie Steele:* no consensus on any bar afaik.

</details>

File: index.html [hunk 1]
```diff
@@ -3281,6 +3281,13 @@ <h3>Reserved Extension Points</h3>
 reserved property. For more information related to adding `type` information,
 see Section <a href="#types"></a>.
         </p>
+        <p class="issue" data-number="882">
```
Comment (msporny) [inline index.html:3284]:
```suggestion
        <p class="issue" title="Extension points under consideration by the Working Group">
```

Comment (msporny) [reply index.html:3284]: I'd be a +1 to this PR if the change above is made.

Comment (decentralgabe) [reply index.html:3284]: is there a way to have both title and data number?

**Review by decentralgabe — APPROVED** [review]

**Comment by OR13** [comment]
@msporny please re-review, I have merged your change suggestion.

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**